.gitignore

```diff
@@ -1,2 +1,4 @@
 *.o
-os-update-minui
+RPMS
+yamui
+screensaverd
```

Comment:
IMO "screensaverd" is a bit too generic sounding name for this kind of special circumstances utility.